src/cmd/serve.rs

```diff
@@ -196,6 +199,11 @@ pub fn serve(interface: &str, port: &str, output_dir: &str, config_file: &str) -
                                 console::info(&format!("-> Sass file changed {}", path.display()));
                                 rebuild_done_handling(&broadcaster, site.compile_sass(&site.base_path), &p);
                             },
+                            (ChangeKind::Config, _) => {
+                                console::info(&format!("-> Config changed {}", path.display()));
```

Comment:
there is only one config file, no need to show the path. The message should say that the whole site will be reloaded and will need to refresh the browser